addons/knobs/src/KnobManager.test.js

```diff
@@ -7,37 +7,21 @@ describe('KnobManager', () => {
       const testManager = new KnobManager();
 
       beforeEach(() => {
-        testManager.knobStore = {
-          set: jest.fn(),
-          get: () => ({
-            defaultValue: 'default value',
-            value: 'current value',
-            name: 'foo',
-          }),
-        };
+        testManager.knobStore = { set: jest.fn(), get: () => ({ defaultValue: 'default value', value: 'current value', name: 'foo' }) };
```

Comment:
@ndelangen what's going on here? any chance we fix the linting setup so that our code is not filled with non-functional changes like this again and again? i think this is probably the fifth PR that i've looked at over the past few months that contains hundreds of lines of changes that have nothing to do with the functionality of the code. it looks really pretty when you are generating visualizations of the codebase, but wastes a lot of time for PR review and i've heard from others who are also annoyed about this. curious to hear your thoughts.